view/templates/intros.tpl

```diff
@@ -9,7 +9,7 @@
 {{if $gender}}<dl><dt>{{$gender_label}}</dt><dd>{{$gender}}</dd></dl>{{/if}}
 {{if $keywords}}<dl><dt>{{$keywords_label}}</dt><dd>{{$keywords}}</dd></dl>{{/if}}
 {{if $about}}<dl><dt>{{$about_label}}</dt><dd>{{$about}}</dd></dl>{{/if}}
-<div class="intro-knowyou">{{$knowyou}}</div>
+<div class="intro-knowyou">{{$lbl_knowyou}} {{$knowyou}}</div>
```

Comment:
You could rename "about_label" (and the other ones) if you like. But the mechanism should be unique. I'm unsure how it is done in other templates. Is it more "*_label" or more "lbl_*" or a third form?